### ISSUE TITLE
White screen of death at launch

### ISSUE THREAD

**Issue by matheusfillipe** [thenewboston-developers/Account-Manager#480]
**Bug Description**
The version 29 of the app is failing to properly launch on linux. Only a white screen is displayed and the top menu. Version 28 was working fine.

**Steps to Reproduce**
`./TNB-Account-Manager-1.0.0-alpha.29-linux.AppImage `

I tried resizing, applying zoom and resting it... (some render problems related to nvidia proprietary drivers sometimes causes glitches that can be solved like that, but in this case seems nothing is being rendered at all)

![image](https://user-images.githubusercontent.com/24435787/99634569-7d54e300-2a1f-11eb-9d5f-e1d3e6cedbf0.png)

**OS and Browser**
OS: Arch Linux x86_64, DE: KDE Plasma 5.20.2, CPU: Intel i7-7700HQ (8) @ 3.800GHz, GPU: NVIDIA GeForce GTX 1050 Ti Mobile, GPU: Intel HD Graphics 630, Memory: 15.5 GiB

**Shell Output**

```sh
$ ./TNB-Account-Manager-1.0.0-alpha.29-linux.AppImage       
(node:1903634) ExtensionLoadWarning: Warnings loading extension at /home/matheus/.config/TNB Account Manager/extensions/fmkadmapgofadopljbjfkapdkoienihi: Unrecognized manifest key 'browser_action'. Unrecognized manifest key 'minimum_chrome_version'. Unrecognized manifest key 'update_url'. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system. 
(node:1903634) ExtensionLoadWarning: Warnings loading extension at /home/matheus/.config/TNB Account Manager/extensions/lmhkpmbekcpmknklioeibfkpmmfibljd: Unrecognized manifest key 'commands'. Unrecognized manifest key 'homepage_url'. Unrecognized manifest key 'page_action'. Unrecognized manifest key 'short_name'. Unrecognized manifest key 'update_url'. Permission 'notifications' is unknown or URL pattern is malformed. Permission 'contextMenus' is unknown or URL pattern is malformed. Permission 'tabs' is unknown or URL pattern is malformed. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system. 
Added Extension: Redux DevTools

```
This output is exactly the same with version 28 thought so wont probably be of much helpful.

**Account Number**

`72d31b3aa84e4a0cf58dc5527b5a0a54a4499716994a2f7c0507e32b9985156c`

**Comment by dstekanov** [comment]
The same on mac. But in console, I see an error: "Uncaught Error: Project version could not be inferred. Please specify the `projectVersion` option."

**Comment by spconway** [comment]
Same issue on Windows 10 Home v2004 build 19041.630 using version 29 of alpha.

**Comment by lapstjup** [comment]
Should be resolved by #484

**Comment by matheusfillipe** [comment]
I can confirm is working now on linux with alpha-30.
